Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -317,7 +317,21 @@ Examples of commands:
 .. index:: DataHandler; Data array
 .. _tce-data:
 
-Data array
+Accessing the uid of copied records:
+------------------------------------
+
```

Comment:
I'd first start with an explanation of `copyMappingArray_merged` here, when it is filled, how it's structure looks like, etc. Then the following example is way easier to understand then just :man_shrugging:

Reply:
Well, it's pretty much 🤷‍♂️ to me as well, as I just stumbled upon it. 😄 

I added my observations  and a warning about `copyMappingArray`.